src/lib.rs

```diff
@@ -590,6 +592,52 @@ impl<T: Debug + PrimInt + One + Zero> Vob<T> {
         }
     }
 
+    /// Append all elements from a second Vob to this Vob.
```

Comment:
I think we need to say *why* you might want to use this function over just `extend` (saying something along the lines of "This is much faster than `extend`" is probably enough).

Reply:
I've clarified this.